### ISSUE TITLE
Adafruit motor hat focuser not working after latest Astroberry upgrade

### ISSUE THREAD

**Issue by ShianMull** [rkaczorek/astroberry-server#197]
Hi,

I’m hoping someone can help me with my Adafruit Motorhat focuser not working after latest update/upgrade?

The driver is in my Indi folders and equipment config OK, but it does not appear to connect successfully when starting EKOS (other drivers are working fine).

I recently updated the Astroberry to the latest and it is following this that the previously reliable focuser is no longer connecting and selectable in the focuser tab of EKOS.

Any help very much appreciated.

All the best

Mark

**Comment by ShianMull** [comment]
A further update - I tried changing for another focusser that I have, and when I then looked to edit back to the Motor Hat Focuser, it was no longer a menu option!

I’m probably missing something obvious, but I can’t for the life of me see what is wrong.

Any help gratefully received.

All the best

Mark

**Comment by ShianMull** [comment]
Update that issue is resolved - I have now got the focuser working by installing the Astroberry-amh driver from the GitHub repository and using cmake.

Not sure what happened, but in updating/upgrading my Astroberry, the motorhat Indi driver was removed - don’t know if this was an error in my update process or the driver was not in the latest update.  Probably worth checking the latest build implementation?

All the best

Mark
